--- /home/znmeb/local/src/pdq/Makeall	2011-03-07 18:52:08.000000000 -0800
+++ Makeall	2012-11-16 13:25:35.375385253 -0800
@@ -11,7 +11,7 @@
 (cd lib; make)
 (cd perl5; ./setup.sh)
 (cd python; make)
-(cd R; cp ../lib/*.[ch] pdq/src; cp ../lib/P*.[h] pdq/lib; R CMD INSTALL pdq)
+(cd R; cp ../lib/*.[ch] pdq/src; cp ../lib/P*.[h] pdq/lib)
 
 #---------------------------------------------------------------------
 
