--- /usr/local/src/rstudio-rstudio-e65a459/dependencies/common/install-boost	2013-03-12 11:24:46.000000000 -0700
+++ ./install-boost	2013-03-19 19:58:19.916770466 -0700
@@ -35,12 +35,7 @@
 then
    # download boost
    BOOST_URL=http://sourceforge.net/projects/boost/files/boost/$BOOST_VERSION_NUMBER/$BOOST_TAR/download?use_mirror=autoselect
-   if [ "$PLATFORM" == "Darwin" ]
-   then
-      curl -L $BOOST_URL > $BOOST_TAR
-   else
-      wget $BOOST_URL -O $BOOST_TAR
-   fi
+   curl -k -L $BOOST_URL > $BOOST_TAR
    
    # remove existing boost installation
    sudo rm -rf $BOOST_DIR
