--- install-gwt	2012-06-28 05:39:51.000000000 -0700
+++ /data/znmeb/Projects/Computational-Journalism-Server/Base/install-gwt	2012-06-23 18:39:35.085359341 -0700
@@ -23,12 +23,7 @@
 
 download()
 {
-  if [ "$PLATFORM" == "Darwin" ]
-  then
-    curl -L https://s3.amazonaws.com/rstudio-buildtools/$1 > $1
-  else
-    wget https://s3.amazonaws.com/rstudio-buildtools/$1 -O $1
-  fi
+  curl -k -L https://s3.amazonaws.com/rstudio-buildtools/$1 > $1
 }
 
 # target directory for gwt
