--- /home/znmeb/local/src/rstudio-rstudio-784d988/src/cpp/CMakeLists.txt	2012-10-13 17:39:30.000000000 -0700
+++ CMakeLists.txt	2012-10-31 14:59:10.051806178 -0700
@@ -113,7 +113,7 @@
 # UNIX BOOST
 if(UNIX)
    # prefer static link to our custom built version
-   set(RSTUDIO_TOOLS_BOOST /opt/rstudio-tools/boost/boost_1_50_0)
+   set(RSTUDIO_TOOLS_BOOST ${HOME}/local/rstudio-tools/boost/boost_1_50_0)
    if(EXISTS ${RSTUDIO_TOOLS_BOOST})
       # find headers
       set(Boost_USE_STATIC_LIBS ON)
