--- /home/znmeb/local/src/rstudio-rstudio-784d988/dependencies/common/install-boost	2012-10-13 17:39:30.000000000 -0700
+++ install-boost	2012-10-31 13:31:04.827993607 -0700
@@ -24,7 +24,7 @@
 # vars
 BOOST_VERSION_NUMBER=1.50.0
 BOOST_VERSION=boost_1_50_0
-BOOST_DIR=/opt/rstudio-tools/boost/$BOOST_VERSION
+BOOST_DIR=${HOME}/local/rstudio-tools/boost/$BOOST_VERSION
 BOOST_TAR=$BOOST_VERSION.tar.bz2
 BOOST_BUILD_DIR=boost-build
 
@@ -33,15 +33,10 @@
 then
    # download boost
    BOOST_URL=http://sourceforge.net/projects/boost/files/boost/$BOOST_VERSION_NUMBER/$BOOST_TAR/download?use_mirror=autoselect
-   if [ "$PLATFORM" == "Darwin" ]
-   then
-      curl -L $BOOST_URL > $BOOST_TAR
-   else
-      wget $BOOST_URL -O $BOOST_TAR
-   fi
+   curl -L $BOOST_URL > $BOOST_TAR
    
    # remove existing boost installation
-   sudo rm -rf $BOOST_DIR
+   rm -rf $BOOST_DIR
   
    # untar source (remove existing)
    rm -rf $BOOST_BUILD_DIR
@@ -53,18 +48,12 @@
    cd $BOOST_VERSION
    ./bootstrap.sh  
    
-   # special variation of build for osx (multi-arch, tied to 10.5)
-   if [ "$PLATFORM" == "Darwin" ]
-   then
-     sudo ./bjam --prefix=$BOOST_DIR toolset=clang variant=release threading=multi link=static install
-   else
-     sudo ./bjam --prefix=$BOOST_DIR variant=release install
-   fi
+   ./bjam --prefix=$BOOST_DIR variant=release install
 
    # go back to the original install dir and remove build dir
    cd $INSTALL_DIR
-   sudo rm -rf $BOOST_TAR
-   sudo rm -rf $BOOST_BUILD_DIR
+   rm -rf $BOOST_TAR
+   rm -rf $BOOST_BUILD_DIR
 
 else
 
